src/app/containers/top-right-menu/top-right-menu.component.html

```diff
@@ -1,12 +1,46 @@
-<ng-container *ngIf="menuItems$ | async as menuItems">
-  <p-menu [styleClass]="styleClass || 'alg-top-right-menu'" #menu [popup]="true" [model]="menuItems" appendTo="body"></p-menu>
-  <div class="user-container" *ngIf="userLogin$ | async as userLogin">
-    <span class="alg-secondary-color user-login" *ngIf="!(isNarrowScreen$ | async)">{{ userLogin }}</span>
+<ng-container *ngrxLet="{ userLogin: userLogin$, isNarrowScreen: isNarrowScreen$ } as data">
```

Comment:
I think that's equivalent to: 
```
@let userLogin = userLogin$ | async;
@let isNarrowScreen = isNarrowScreen$ | async;
```
and then using no "ng-container" and using `userLogin` and `isNarrowScreen` directly. 

I think that's more readable and more aligned with current Angular practices.